test/util/velero/velero_utils.go

```diff
@@ -931,14 +939,15 @@ func DeleteBackupResource(ctx context.Context, veleroCLI string, backupName stri
 	return nil
 }
 
-func GetBackup(ctx context.Context, veleroCLI string, backupName string) (string, string, error) {
-	args := []string{"backup", "get", backupName}
+func GetBackup(ctx context.Context, veleroCLI, backupName, veleroNamespace string) (string, string, error) {
+	args := []string{"--namespace", veleroNamespace, "backup", "get", backupName}
```

Comment:
This was added so it can run without failing locally when there is a saved velero config in home directory. Perhaps something that e2e should be able to turn off via a flag `--ignore-home-config`
```
❯ cat ~/.config/velero/config.json
{"namespace":"some-custom-ns"}
```